src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -264,6 +297,35 @@ public void deleteDataset(Integer datasetId) {
         dataSetDAO.deleteDataSets(idList);
     }
 
+    public void deleteDataset(Integer dataSetId, Integer dacUserId) throws IllegalStateException {
+        try {
+            dataSetDAO.begin();
```

Comment:
Not for this PR, but I found that this has problems when actually used. We should be doing this inside a handle instead (see https://jdbi.org/#_handle). Need to file a story once this PR is merged that covers updating this method.